lib/interactor.rb

```diff
@@ -163,4 +163,28 @@ def call
   # Returns nothing.
   def rollback
   end
+
+  private
+
+  # Internal: Determine what keyword arguments (if any) should be passed to the
+  # "call" instance method when invoking an Interactor. The "call" instance
+  # method may accept any number of keyword arguments. This method will extract
+  # values from the context in order to populate those arguments based on their
+  # names.
+  #
+  # Returns an Array of arguments to be applied as an argument list.
+  def arguments_for_call
+    positional_arguments = []
+    keyword_arguments = {}
+
+    method(:call).parameters.each do |(type, name)|
```

Comment:
i think should be here: ```self.class.instance_method(:call)``` instead of ```method(:call)```, because for some reason  method(:call) returns the parameters from the class method "call", and not from the instance method "call" .  I know this because i actually tried this and the only way i could make it work was to use `self.class.instance_method` . 
Or maybe i am missing something.